docs/chapter_phil/What is probability? Bayes contra Hume.rst

```diff
@@ -1,3 +1,801 @@
+each possible value (`Kruschke, Aguinis, & Joo, 2012`_). Now you could use 
+your updated belief as a new prior and repeat the whole process to update your 
+knowledge once more.
+
+Conclusion Bayes
+================
+
+I guess this was a lot to think about so let us take a breath and revisit the 
+concepts. Using Bayesian methods and therefore the subjective probability 
+approach is a way to update your subjective beliefs by combining your belief 
+about a hypothesis and the evidence, and all this with distributions or 
+different probabilities of possible results. This is much more complex than a 
+possible black and white answer where you reject or do not reject a hypothesis. 
+But we have seen that the prior is a double edged sword. It helps us to use 
+previous knowledge (and often we have knowledge on things) but it can have an 
+influence on our results because our previous knowledge might be very wrong 
+and so we might choose a wrong prior. Given enough data this problem might not 
+be so relevant but still it has been opening up debates in science for quite 
+some time (e.g. `Gelman, 2009`_; `Kruschke, 2010`_; `van de Schoot et al. 
+2014`_; `Vanpaemel, 2010`_; `Winkler, 1967`_). Furthermore, the distributions 
+of posterior probabilities might give a more complex picture of reality but 
+often we are forced to make black and white decisions (decide if we want to 
+pay for a medication or not) because we have to act. It is a different approach 
+to probability and now you have heard of it as well. So let us end with some 
+final thoughts.
+
+
+Conclusion chapter
+******************
+
+At this point I hope you have a better understanding of two points of view of 
+probability that are common in the social and behavioral sciences. Of course 
+there is much more to it; more formulas, more mathematics, and different 
+statistical approaches but my goal was to give you a first idea of the concepts 
+that are at the basis of so many different methods in statistics. Both points 
+of view come with their own strengths, weaknesses and possible pitfalls. 
+I do not want to argue for one or against the other but my hope is that you 
+will understand that both points of view have a different aim, a different 
+inference, and are sensitive to different things. You should be aware of your 
+research question and the kind of probability that helps you to find an answer 
+to this question. Do you need a black and white answer using objective 
+probability or do you need a continuous distribution of posterior beliefs using 
+subjective probability? Both probabilities come with a huge toolbox of 
+applicable statistical methods (and some of them are discussed by my colleagues 
+in this volume) and many of those methods can be used with both approaches. So 
+chose your tool and scientific approach to each question you ask very careful 
+and aware of the alternatives. I wish you a pleasant journey into the wonderful 
+world of statistics.
```

Comment:
I don't like this last sentence :) 1. this is not a statistics book, 2. the tone is a bit too much. Maybe just cut it.